src/main/java/com/toy/board/springbootboard/controller/PostController.java

```diff
@@ -0,0 +1,55 @@
+package com.toy.board.springbootboard.controller;
+
+import com.toy.board.springbootboard.ApiResponse;
+import com.toy.board.springbootboard.model.dto.PostDto;
+import com.toy.board.springbootboard.model.service.PostService;
+import javassist.NotFoundException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class PostController {
+
+    private final PostService postService;
+
+    @ExceptionHandler
+    private ApiResponse<String> exceptionHandle(Exception exception) {
+        return ApiResponse.fail(HttpStatus.INTERNAL_SERVER_ERROR.value(), exception.getMessage());
+    }
+
+    @ExceptionHandler(NotFoundException.class)
+    private ApiResponse<String> notFoundHandle(NotFoundException exception) {
+        return ApiResponse.fail(HttpStatus.NOT_FOUND.value(), exception.getMessage());
+    }
+
+    @PostMapping("/posts")
+    public ApiResponse<Long> save(@RequestBody PostDto postDto) throws NotFoundException {
+        return ApiResponse.ok(postService.save(postDto));
+    }
+
+    @GetMapping("/posts")
+    public ApiResponse<Page<PostDto>> getAll(Pageable pageable) {
+        Page<PostDto> page = postService.findPosts(pageable);
+        return ApiResponse.ok(page);
+    }
+
+    @GetMapping("/posts/{id}")
+    public ApiResponse<PostDto> get(@PathVariable long id) throws NotFoundException {
+        PostDto postDto = postService.findById(id);
+        return ApiResponse.ok(postDto);
+    }
+
+    @PostMapping("posts/{id}")
```

Comment:
```suggestion
    @PostMapping("/posts/{id}")
```
여기는 `/`가 빠졌네요 😯 모두 Posts가 공통된다면, RequestMapping 어노테이션으로 컨트롤러 상단에 공통으로 가져갈 수 있습니다.